libndt.hpp

```diff
@@ -1960,8 +2017,37 @@ bool Client::ndt7_upload() noexcept {
     constexpr auto measurement_interval = 0.25;
     std::chrono::duration<double> interval = now - latest;
     if (interval.count() > measurement_interval) {
+      nlohmann::json measurement;
+      measurement["AppInfo"] = nlohmann::json();
+      measurement["AppInfo"]["ElapsedTime"] = elapsed.count();
```

Comment:
The measurement unit here should be microseconds. Is that the case?

Reply:
It was not, but it is now. I've noticed the JSON value received from the server during a download test is in seconds, though:
```
"elapsed":0.750120832
```

Is that expected? However, the client now prints (and sends to the server) microseconds during the upload test.

Reply:
Yes, it is expected. We changed the spec and v0.8.0 uses microsecond because that is what every other gauge uses as measurement unit. Yet, we've not released a ndt-server version using this new format. I'm leaning towards implementing here v0.8.0, because otherwise we'd need to change the code again in a few days. Okay?

Reply:
Yup. That makes this client future-proof, or at least up-to-date with the spec. :)